src/components/Cart/Cart.js

```diff
@@ -8,36 +7,30 @@ const Cart = (props) => {
   const deliveryFee = 4.99;
 
   const [selectedItems, setSelectedItems] = useState([]);
-  const [totalCost, setTotalCost] = useState();
-  const [subTotalCost, setSubTotalCost] = useState();
-  const [taxes, setTaxes] = useState();
-  const [isChecked, setIsChecked] = useState(false); // New state for checkbox checked status
+  const [totalCost, setTotalCost] = useState(0);
+  const [subTotalCost, setSubTotalCost] = useState(0);
+  const [taxes, setTaxes] = useState(0);
+  const [isChecked, setIsChecked] = useState(false);
 
   useEffect(() => {
     let subTotal = 0;
 
-    cartItems?.price?.map((item) => {
-      subTotal = subTotal + Number(item);
-      console.log(item);
-      return subTotal;
+    cartItems?.forEach((item) => {
+      subTotal += Number(item.price);
     });
 
-    // Calculate taxes and total cost
-    const calculateCost = () => {
-      setTaxes(subTotalCost * 0.06);
-      setTotalCost(4.99 + taxes + subTotalCost);
-    };
-
-    calculateCost();
-  }, [cartItems, subTotalCost, taxes]);
+    setSubTotalCost(subTotal);
+    setTaxes(subTotal * 0.06);
+    setTotalCost(deliveryFee + subTotal + (subTotal * 0.06));
```

Comment:
you should be using taxes here instead of the (subTotal * 0.06)

Reply:
like this? setTotalCost(deliveryFee + subTotal + (subTotal * taxes)); ?

Reply:
I did it and tested it and it wasn't quite right. Two books selling for $14.99 came up to $62. Looks like it's calculating the taxes per item individually and adding a separate shipping and then doing the calculations for the total cost. I need to figure it out before I push this unless I was wrong in what you were asking me to change.

Reply:
deliveryFee + taxes + subTotal